docs/UsersGuide/source/RunSRW.rst

```diff
@@ -534,6 +535,77 @@ Valid values for configuration variables should be consistent with those in the
 
 To configure an experiment and python environment for a general Linux or Mac system, see the :ref:`next section <LinuxMacEnvConfig>`. To configure an experiment to run METplus verification tasks, see :numref:`Section %s <VXConfig>`. Otherwise, skip to :numref:`Section %s <GenerateWorkflow>` to generate the workflow.
 
+.. _PlotOutput:
+
+Plot the Output
+-----------------
+
+An optional Python plotting task (PLOT_ALLVARS) can be activated in the workflow to generate plots for the :term:`FV3`-:term:`LAM` post-processed :term:`GRIB2`
+output over the :term:`CONUS`. It generates graphics plots for a number of variables, including:
+
+   * 2-m temperature
+   * 2-m dew point temperature
+   * 10-m winds
+   * 250 hPa winds
+   * Accumulated precipitation
+   * Composite reflectivity
+   * Surface-based :term:`CAPE`/:term:`CIN`
+   * Max/Min 2-5 km updraft helicity
+   * Sea level pressure (SLP)
+
+.. COMMENT: * 500 hPa heights, winds, and vorticity --> seems to be omitted? Why?
+
+This workflow task can produce both plots from a single experiment and difference plots that compare the same cycle from two experiments. When plotting the difference, the two experiments must be on the same domain and available for 
+the same cycle starting date/time and forecast hours. Other parameters may differ (e.g., the experiments may use different physics suites).
+
+.. _Cartopy:
+
+Cartopy Shapefiles
+^^^^^^^^^^^^^^^^^^^^^
+
+The Python plotting tasks require a path to the directory where the Cartopy Natural Earth shapefiles are located. The medium scale (1:50m) cultural and physical shapefiles are used to create coastlines and other geopolitical borders on the map. On `Level 1 <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`__ systems, this path is already set in the system's machine file using the variable ``FIXshp``. Users on other systems will need to download the shapefiles and update the path of ``$FIXshp`` in the machine file they are using (e.g., ``$HOME/ush/machine/macos.yaml`` for a generic MacOS system, where ``$HOME`` is the path to the ``ufs-srweather-app`` directory). The subset of shapefiles required for the plotting task can be obtained from the `SRW Data Bucket <https://noaa-ufs-srw-pds.s3.amazonaws.com/NaturalEarth/NaturalEarth.tgz>`__. The full set of medium-scale (1:50m) Cartopy shapefiles can be downloaded `here <https://www.naturalearthdata.com/downloads/>`__. 
```

Comment:
@gspetro-NOAA Would it be best to replace `$HOME` with `$SRW` in this line?  The `$SRW` environment variable should already be set in the BuildSRW chapter, so it should already be created before the RunSRW chapter.